docs/contracts/overview/DigitalAssets.md

```diff
@@ -12,6 +12,12 @@ The **Digital Asset (Token and NFT 2.0)** contracts are the newest advanced vers
 
 ## Comparisons with ERC20 / ERC721
 
+:::danger beware
+
+The LSP7 contracts Compatible to ERC20 and LSP8 contracts Compatible with ERC721 are being deprecated and will be deleted from [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts) repository. Howether if you want to stil use/maintain them, they will remain available in the version [`0.14.0`](https://github.com/lukso-network/lsp-smart-contracts/releases/tag/lsp-smart-contracts-v0.14.0).
```

Comment:
compatible _to_ or compatible _with_ ? You are using both here.

Why uppercase on the word Compatible?